study.md

```diff
@@ -40,14 +42,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+It means you are permenetly changing my_string to have capital letters
 ```
 
 ## Instantiation
 How do you create an instance of a class in Ruby?
 
 ```ruby
-# your answer here
+an instance is a variable you set to be available within a class
```

Comment:
That's a class variable or an instance variable.  We want to know how to create an instance of an existing class.  `class_instance = ClassName.new`